docs/resources/service.md

```diff
@@ -24,12 +24,17 @@ The following configuration starts a Docker Service with
 - moreover, uses the `container` runtime
 
 ```terraform
+resource "docker_image" "foo" {
+  name         = "repo.mycompany.com:8080/foo-service:v1"
+  keep_locally = true
+}
+
 resource "docker_service" "foo" {
   name = "foo-service"
 
   task_spec {
     container_spec {
-      image = "repo.mycompany.com:8080/foo-service:v1"
+      image = docker_image.foo.latest
```

Comment:
is there a way to avoid confusion between the image's `:latest` tag and the `.latest` attribute, which in this case does not appear to be the same.

Reply:
yes, you're right. This is confusing. I added a more extended description to it but we will also revise it in #161

Reply:
The description got updated. we might remove the `latest` attribute and replace it with a clearer one like `sha256_digest`

Reply:
yep, that'd be much better.  :)

Reply:
added in 812ae724f2045cf38b68917f82dec443c764cfd2

Reply:
As mentioned in the top-level README.md comment, we probably need to provide the `docker_service.task_spec.image_spec.image` attribute either a full `repo_digest` string or the usual `<registry>/<image>[:<tag>]` so that other swarm nodes that the provider's not connected to know what they need to pull.